Sources/IBLinterKit/Rules/CustomModuleRule.swift

```diff
@@ -50,9 +50,20 @@ extension Rules {
                 }
             }
 
+            func resolvePath(_ includeURLString: String) -> URL {
+                if #available(OSX 10.11, *) {
+                    let url = URL(fileURLWithPath: includeURLString, relativeTo: context.workDirectory)
+                    return url.standardizedFileURL
+                } else {
+                    fatalError("Unsupported OS version")
```

Comment:
Currently, SwiftPM can't specify the target SDK version. It generates as 10.10. So this condition is needed.

I think we need not support such an ancient OS 🦖

Reply:
Since Swift 5, we can specify SDK version 🎉 
https://github.com/IBDecodable/IBLinter/pull/95/commits/ce3ba13fa4873c914dd3b6c9a223a6ba05743b5b

Thanks @omochi